db/migrate/20150910010112_create_team_memberships.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTeamMemberships < ActiveRecord::Migration
+  def change
+    create_table :team_memberships do |t|
+      t.belongs_to :team
+      t.belongs_to :user
+      t.integer :role, default: 0
```

Comment:
Check in the model - the :role field is an [ActiveRecord enum](http://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html). I've got :user and :role roles defined in the model which gives us magic helpers like `user.admin?` straightaway. Also, using an enum instead of a boolean gives more flexibility should we need to add more roles in the future.